### PR TITLE
Xton has a dz60

### PR DESCRIPTION
**Pull request by xton**
Here's my keymap ported to it!

...and also a few bugfixes to my niu_mini layout (sorry).

### CODE REVIEW

File: keyboards/dz60/keymaps/xtonhasvim/keymap.c [hunk 1]
```diff
@@ -0,0 +1,94 @@
+#include "dz60.h"
```
Comment (mechmerlin) [inline keyboards/dz60/keymaps/xtonhasvim/keymap.c]: You can replace `#include "dz60.h" ` with `#include QMK_KEYBOARD_H`

File: keyboards/dz60/keymaps/xtonhasvim/keymap.c [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+enum layers {
+  _QWERTY,
+  _FUN,
+  _MOVE,
+  _MOUSE
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	[_QWERTY] = LAYOUT(
```
Comment (mechmerlin) [inline keyboards/dz60/keymaps/xtonhasvim/keymap.c]: This looks to be a standard 60% layout. You can use LAYOUT_60_ansi and you won't need to add KC_NO to your keymap.

Comment (drashna) [reply keyboards/dz60/keymaps/xtonhasvim/keymap.c]: Yup, the board supports it: 
https://github.com/qmk/qmk_firmware/blob/master/keyboards/dz60/dz60.h#L94

File: keyboards/dz60/keymaps/xtonhasvim/keymap.c (hunk 1 above)
Comment (drashna) [reply keyboards/dz60/keymaps/xtonhasvim/keymap.c]: ^

(OPTIONAL)  Also, you could probably drop the keymap into /layouts/community/60_ansi/(username)/ instead. That way, it's usable with any compatible 60% ansi board.

File: keyboards/dz60/keymaps/xtonhasvim/keymap.c (hunk 2 above)
Comment (xton) [reply keyboards/dz60/keymaps/xtonhasvim/keymap.c]: Except that I've got a split spacebar and `LAYOUT_60_ansi()` assumes not that.

File: keyboards/dz60/keymaps/xtonhasvim/keymap.c (hunk 1 above)
Comment (xton) [reply keyboards/dz60/keymaps/xtonhasvim/keymap.c]: I'd worry about the LED handling though. That doesn't look very standard? I'm using the capslock light to show when you're in VIM mode.

File: keyboards/dz60/keymaps/xtonhasvim/keymap.c (hunk 2 above)
Comment (drashna) [reply keyboards/dz60/keymaps/xtonhasvim/keymap.c]: Ah, okay, then you should be fine, here.

File: keyboards/dz60/keymaps/xtonhasvim/keymap.c (hunk 1 above)
Comment (drashna) [reply keyboards/dz60/keymaps/xtonhasvim/keymap.c]: For which part? 

The differently includes basically do the same thing, but ... we're trying to move to a more hardware agnostic approach. 

So, given the rest, leaving it here is fine, but changing the include is still recommended.

Comment (xton) [reply keyboards/dz60/keymaps/xtonhasvim/keymap.c]: Cool. Include is changed. I was worried about how the LED backlights are handling. In the default layout it looks like this:

**Comment by drashna** [comment]
Thanks!

**Pull request merged**